airbyte-integrations/connectors/source-s3/integration_tests/config_minio.json

```diff
@@ -6,7 +6,7 @@
     "aws_access_key_id": "123456",
     "aws_secret_access_key": "123456key",
     "path_prefix": "",
-    "endpoint": "http://10.0.40.43:9000"
+    "endpoint": "http://100.89.162.14:9000"
```

Comment:
Why did you have to make this change?

Reply:
ah this wasn't done manually, but still the change in unnecessary, I will revert it

Reply:
reverted